### PR TITLE
Copy atom_absolute extension from WTD and rewrite multiple categories

### PR DESCRIPTION
**Pull request by ericholscher**

### CODE REVIEW

File: exts/atom_absolute.py [hunk 1]
```diff
@@ -7,49 +7,56 @@
...
+    rewritten_feed_path = os.path.join(
+        app.outdir, 'blog/archive/category/{tag}/atom_absolute.xml'.format(tag=tag)
+    )
 
-        # Rewrite the namespace map to not be Null
-        namespace = list(root.nsmap.values())[0]
-        nsmap = {'atom': namespace}
+    if not os.path.isfile(feed_path):
+        logger.error('Atom feed does not exist at: {}'.format(feed_path))
+        return
```
Comment (humitos) [inline exts/atom_absolute.py:40]: Shouldn't we raise an exception here so the build fails and we notice it?

File: exts/atom_absolute.py [hunk 2]
```diff
@@ -7,49 +7,56 @@
...
+        html_content = html.fromstring(content.text)
+        # Remove janky system messages
+        for to_remove in html_content.find_class('system-message'):
+            to_remove.drop_tree()
+        # fix links
+        html_content.make_links_absolute(feed_url)
+        content.text = html.tostring(html_content)
 
-def setup(app):
-    app.connect('build-finished', rewrite_atom_feed)
```
Comment (humitos) [inline exts/atom_absolute.py:49]: Don't we need to keep this as an extension?

**Review by humitos — COMMENTED** [review]
I'm not sure to understand how this works. Is the idea behind this is to remove `newsletter` and `changelog` entries from the XML file?

**Comment by ericholscher** [comment]
This was harder to get working than I hoped, so gonna hold off for now.

**Pull request closed without merging**